src/routes/(mathex)/mathex/app/play/[id]/+page.svelte

```diff
@@ -150,6 +150,8 @@
             return;
           }
           socket.emit("answer", answer);
+          document.getElementById(this.id).style.setAttribute("disabled", "");
+          setTimeout( { document.getElementById(this.id).style.removeAttribute("disabled"); } , 16000);
```

Comment:
![image](https://github.com/user-attachments/assets/9f4492b8-aebf-491f-adda-cc1d5847653a)
This code has a syntax error